apps/studio/src/main.ts

```diff
@@ -22,6 +22,7 @@ import './assets/styles/app.scss'
 import $ from 'jquery'
 
 import 'codemirror/mode/sql/sql'
+import 'codemirror/mode/javascript/javascript'
```

Comment:
Was this causing errors or something?

Reply:
Based on my digging, they dont have a specific language for JSON, although you can use the JS mode to act like JSON.
If this isnt true, then please enlighten me, but that is atleast the info I could find online.

Reply:
No I'm sure that's good! I just wasn't sure if it was just a stray import or not